src/OrchardCore/OrchardCore.Email.Abstractions/Extensions/EmailServiceExtensions.cs

```diff
@@ -0,0 +1,54 @@
+using System.Threading.Tasks;
+
+namespace OrchardCore.Email;
+
+/// <summary>
+/// Provides an extension methods to <see cref="ISmtpService"/>.
+/// </summary>
+public static class EmailServiceExtensions
+{
+    /// <summary>
+    /// Sends the specified message to an SMTP server for delivery.
+    /// </summary>
+    /// <param name="emailService">The <see cref="IEmailService"/>.</param>
+    /// <param name="to">The email recipients.</param>
+    /// <param name="subject">The email subject.</param>
+    /// <param name="body">The email body.</param>
+    /// <param name="isHtmlBody">Whether the <paramref name="body"/> in HTML format or not. Defaults to <c>true</c>.</param>
+    /// <param name="cc">The carbon copy emails. Defaults to <c>null</c>.</param>
+    /// <param name="bcc">A blind copy emails. Defaults to <c>null</c>.</param>
+    /// <param name="replyTo">The replied to emails. Defaults to <c>null</c>.</param>
+    /// <returns></returns>
+    /// <exception cref="System.ArgumentException"></exception>
+    public static async Task<EmailResult> SendAsync(this IEmailService emailService, string to, string subject, string body, bool isHtmlBody = true, string cc = null, string bcc = null, string replyTo = null)
```

Comment:
I don't think we should add all these extensions. I just don't think we'll used them. How about adding couple constructors to MailMessage instead?

Reply:
I already saw such a pattern a long time ago in many frameworks & CMSs

> How about adding couple constructors to MailMessage instead?

Me as I used I will prefer a simple extension rather than instantiate a new `MailMessage`

Reply:
I think such shortcuts are only useful for the absolutely most frequent use cases. Otherwise, we'll duplicate the API.

This is especially important because unless you deliberately use named arguments, a method call will be less readable than an object initialization. See e.g. below; you have to put more effort into understanding which argument of `SendAsync()` does what, as opposed to obvious names:

![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/537d71da-0f05-427f-8730-8fe6028a2fc6)

Can we say something like "75% percent of `SendAsync()` calls only use `To`, `Subject`, and `Body` from `MailMessage`? Because then it's worthwhile adding an extension for that scenario.

Reply:
You didn't reply here.

Reply:
I wrote an extension for that

```csharp
    public static async Task<EmailResult> SendAsync(this IEmailService emailService, string to, string subject, string body, bool isHtmlBody = true)
    {
        var message = new MailMessage
        {
            To = to,
            Subject = subject,
            Body = body,
            IsHtmlBody = isHtmlBody
        };

        return await emailService.SendAsync(message);
    }
```

Reply:
I'm not convinced it's necessary or even beneficial though. Please address my https://github.com/OrchardCMS/OrchardCore/pull/14772#discussion_r1536077333 comment.

Reply:
> Can we say something like "75% percent of SendAsync() calls only use To, Subject, and Body from MailMessage?

I understand that the extension should have the above arguments, right? Or maybe I missundertood

Reply:
No, it's a question :).

Reply:
I'm out of control because I'm working with many things :)

Yes we can say 80 - 90% uses the above arguments